crates/biome_js_analyze/tests/specs/source/organizeImports/issue-5710.js.snap

```diff
@@ -4,10 +4,33 @@ expression: issue-5710.js
 ---
 # Input
 ```js
-// The last `inject` specifier is a bogus node, so this won't emit any code actions.
+// The last `inject` specifier is a bogus node.
 import { inject, Injectable, inject } from '@angular/core';
 
-// The last `key` attribute is a bogus node, so this won't emit any code actions.
+// The last `key` attribute is a bogus node.
 import { names } from "module-name" with { key: "data", key2: "data2", key: "data3" };
 
 ```
+
+# Diagnostics
+```
+issue-5710.js:2:1 assist/source/organizeImports  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

Comment:
This is a little improvement over the previous implementation :)

Reply:
This is really edgy: you have to have buggy nodes like here where specifiers are duplicated.

Reply:
I think we did this in the past, where we fixed (and emitted diagnostics) from broken nodes, rather than panicking. It's still a good improvement for the DX, even when it's an edge case